src/server/pkg/serviceenv/service_env.go

```diff
@@ -153,7 +153,7 @@ func (env *ServiceEnv) initKubeClient() error {
 			return fmt.Errorf("could not initialize kube client: %v", err)
 		}
 		return nil
-	}, backoff.RetryEvery(time.Second).For(time.Minute))
+	}, backoff.RetryEvery(time.Second).For(5*time.Minute))
```

Comment:
why these increases? 5min seems like a long time to wait for the servers to initialize.

Reply:
Yeah, it should be in general. The main reason for the bump (as far as the pach client is concerned) was to ensure that an object storage client timeout would occur before the pach client timeout (if the object storage client would time out for whatever reason). I don't have a great idea where all the possible points of slowness for startup are now and where they will be in the future, so I figured being conservative at 5 mins would be fine considering this is not something that should happen frequently and is not recoverable. I am open to bumping this back down though if you feel strongly about it.